src/plugins/bitcoin/BitcoinPluginHomepage.tsx

```diff
@@ -0,0 +1,9 @@
+import { RawText } from '../../components/Text'
+
+export function BitcoinPluginHomepage() {
+  return (
+    <div>
+      <RawText>HELLOOOOOOOO</RawText>
```

Comment:
the description of this PR says it's a draft, but it's not marked as one, so IDK if this is actually being proposed to be merged as-is... but for the avoidance of doubt, this is good for a demo but we probably shouldn't merge it.

Reply:
It's a placeholder to show a working example of a plugin.